app/models/concerns/created_by_concern.rb

```diff
@@ -16,4 +16,10 @@ def created_by_user?
   def created_by_prescripteur?
     created_by_type == "Prescripteur"
   end
+
+  def prescription?
+    return true if created_by_prescripteur?
+
+    organisation.in?(created_by.organisations) if created_by_agent?
```

Comment:
Attends, y'a pas un contresens là ? Pour moi, il y a prescription si l'agent qui a créé le RDV n'est **pas** dans l'orga du RDV, et donc ce serait `!in?`, non ?

Et pour la clarté du code, what about this?

```ruby
def created_by_external_agent?
  created_by_agent? && !organisation.in?(created_by.organisations)
end

def prescription?
  created_by_prescripteur? || created_by_external_agent?
end
```

Reply:
oh, bien vu !

Reply:
Merci pour ca 👍